### ISSUE TITLE
Server segfault with bots

### ISSUE THREAD

**Issue by Thann** [opensoldat/opensoldat#11]
I run `soldatserver -bots_random_bravo 1` and immediately get this:

```
...
[NET] Server is listening on 0.0.0.0:23073
[FileServer] Starting fileserver on 0.0.0.0:23083
Shutting down server...
Shutting down game networking.
[NET DEBUG] Shutting down low level socket/threading support.
[FileServer] Stopping fileserver
An unhandled exception occurred at $0000000000470462:
EStringListError: List index (0) out of bounds
  $0000000000470462
  $00000000004386A8
  $000000000042A9C5
  $000000000040363E

An unhandled exception occurred at $000000000041BF47:
EAccessViolation: Access violation
  $000000000041BF47
  $0000000000418FC1
  $000000000040363E
```

**Comment by helloer** [comment]
Unpack `bots` directory from soldat.smod into `configs`

**Comment by Thann** [comment]
So, it doesn't look like the `bots` dir gets added to the smod by `create_smod.sh`, but copying the dir from the base repo fixed the crash!

1. Should I create a PR to add the bots dir?
2. Should I make a PR to look for the bots info in the smod?

**Comment by Shoozza** [comment]
Seems like the server expects to load the bots from the /configs/bots directory but in the base repository they are in /bots instead (inside the server directory).

Moving the bots directory inside the configs folder seems to fix the packaging issue.

I'd suggest to move it there and make a pull request.
Or wait and I'll do it for you tomorrow.

Thanks for the report :)

**Comment by furai** [comment]
Fixed in #12
